part-1/modeling-concepts-modeling-of-proteins.md

```diff
@@ -1,18 +1,18 @@
-# Modeling Concepts - Modeling of Proteins‌
+# Modeling of Proteins‌
 
-Therapeutic proteins are an increasingly important class of drugs. Particularly, monoclonal antibodies are used for different indications including cancer, inflammatory and autoimmune diseases [88]. These engineered antibody fragments with tailored pharmacokinetic properties have the potential to be used as diagnostic and therapeutic agents [[34](../references.md#34)].
+Therapeutic proteins are an increasingly important class of drugs. Particularly, monoclonal antibodies are used for different indications including cancer, inflammatory and autoimmune diseases \[[88](../references.md#88)\]. These engineered antibody fragments with tailored pharmacokinetic properties have the potential to be used as diagnostic and therapeutic agents \[[34](../references.md#34)\].
 
-PK-Sim® offers a model specifically designed to describe the pharmacokinetics of proteins and other macromolecules [[114]](../references.md#114). The pharmacokinetics of protein therapeutics are governed by a number of unspecific and specific processes, which, have to be incorporated to describe the pharmacokinetics of protein therapeutics. This includes [[42](../references.md#42)]:
+PK-Sim® offers a model specifically designed to describe the pharmacokinetics of proteins and other macromolecules [\[114\]](../references.md#114). The pharmacokinetics of protein therapeutics are governed by a number of unspecific and specific processes, which have to be incorporated to describe the pharmacokinetics of protein therapeutics. This includes \[[42](../references.md#42)\]:
```

Comment:
why do you enter `\` before `[` and `]` ?
Also sometimes between the inner brackets, sometimes between the outer brackets